tests/system_tests/test_launch/test_job_status_tracker.py

```diff
@@ -8,11 +8,11 @@
 
 
 @pytest.mark.asyncio
-async def test_check_stop_run_not_exist(wandb_init):
+async def test_check_stop_run_not_exist(user):
```

Comment:
I don't think `user` is needed here

Reply:
you need it everytime you use init, otherwise you don't have a user

Reply:
Ah gotcha. I know it's a common pattern but I hate that pytest fixtures are syntactically indistinguishable from unused variables 👎